gammapy/data/observations.py

```diff
@@ -716,13 +716,15 @@ def ids(self):
         """List of observation IDs (`list`)."""
         return [str(obs.obs_id) for obs in self]
 
-    def select_time(self, time_intervals):
+    def select_time(self, time_intervals, inverted=False):
```

Comment:
We should be consistent in the code base. I think we more frequently use `exclude` as an argument name.

Reply:
I used `inverted` because it is the argument name used in `ObservationTable.select_time_range`. But I agree that we should be consistent in the code base, whether it is `excude` or `inverted`.